facets_overview/proto/BUILD

```diff
@@ -1,10 +1,28 @@
 package(default_visibility = ["//visibility:public"])
 
-load("@io_bazel_rules_closure//closure:defs.bzl", "closure_js_proto_library")
+load("@io_bazel_rules_closure//closure:defs.bzl", "closure_proto_library")
 
 licenses(["notice"])  # Apache 2.0
 
-closure_js_proto_library(
+proto_library(
+    name = "feature_statistics_proto",
+    srcs = [
+        "feature_statistics.proto",
+    ],
+)
+
+closure_proto_library(
+    name = "feature_statistics_closure_proto",
+    deps = [
+        ":feature_statistics_proto",
+    ],
+)
+
+# TODO: Remove when all callers are migrated to '//facets_overview/:feature_statistics_closure_proto'.
+closure_proto_library(
```

Comment:
this should still be "closure_js_proto_library"

Reply:
Done. Thanks!